### PR TITLE
Patch/add redshift

### PR DESCRIPTION
**Pull request by albertobruin**

### CODE REVIEW

File: pkg/config/manager.go [hunk 1]
```diff
@@ -14,6 +14,17 @@ import (
...
 
+type PostgresConnection struct {
+	Name         string `yaml:"name" json:"name"`
+	Username     string `yaml:"username" json:"username"`
+	Password     string `yaml:"password" json:"password"`
+	Host         string `yaml:"host" json:"host"`
+	Port         int    `yaml:"port" json:"port"`
+	Database     string `yaml:"database" json:"database"`
+	PoolMaxConns int    `yaml:"pool_max_conns" json:"pool_max_conns" default:"10"`
+	SslMode      string `yaml:"ssl_mode" json:"ssl_mode" default:"disable"`
```
Comment (karakanb) [inline pkg/config/manager.go:25]: do we need json counterparts for these? I think it's just YAML

File: pkg/postgres/db_test.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	tests := []struct {
+		name      string
+		query     string
+		expected  string
+		setupMock func(mock pgxmock.PgxPoolIface)
+		wantErr   string
+		want      [][]interface{}
+	}{
+		{
+			name:    "Test Select rows",
```
Comment (karakanb) [inline pkg/postgres/db_test.go:27]: nitpick, could you please make these all lowercase to be consistent with all the other tests?

**Review by karakanb — APPROVED** [review]

File: pkg/config/manager.go (hunk 1 above)
Comment (albertobruin) [reply pkg/config/manager.go:25]: Well I'm not sure. The other config types don't have json annotations but they have a `MarshalJSON` [method](https://github.com/bruin-data/bruin/blob/ff74b51e052cf6d0bbd9589af07a61316c9c7be7/pkg/config/manager.go#L72) so I thought it was  necessary?

File: pkg/postgres/db_test.go (hunk 2 above)
Comment (albertobruin) [reply pkg/postgres/db_test.go:27]: sure thing

**Pull request merged**